### ISSUE TITLE
Chef Software products ship with secure, up-to-date, and economically viable OpenJDK instead of insecure and economically prohibitive Oracle JDK

### ISSUE THREAD

**Issue by bish0polis** [chef/chef-server#1692]
Oracle JDK now has incompatible licensing, we committed to not using it anymore by EOY. Replace with OpenJDK everywhere.

[https://blogs.oracle.com/java-platform-group/oracle-java-se-releases-faq](https://blogs.oracle.com/java-platform-group/oracle-java-se-releases-faq)

> The January 15th, 2019 scheduled [Critical Patch Update](https://www.oracle.com/technetwork/topics/security/alerts-086861.html) of Java 8 (8u201, and the related 8u202 [Patch Set Update](https://www.oracle.com/technetwork/java/javase/cpu-psu-explained-2331472.html)) are the last update available under the [BCL license](https://java.com/license)which is generally free for general purpose desktop and server use, and has been the Oracle JDK license for several years. The following update of Java 8, scheduled for April 16, 2019 (8u211 and the related 8u212 Patch Set Update), will be made available under a new license which will be free for personal individual desktop use, and free for development, testing, prototyping and demonstration purposes.

tldr; we can't use Oracle JDK >= 8u211, but all of our products are currently using older versions which are also un-supported at this time.

**product** | **omnibus package** | **version**
-|-|-
chef-backend|server-jre|8u162
chef-server|server-jre|8u162
automate (1)|server-jre|8u162
| |

Aha! Link: https://chef.aha.io/epics/CHEF-E-70

**Comment by sean-horn** [comment]
FYI: The current version of oracle server-jre installed, from Chef Server 12.18.14 to the latest, 13.0.17, is 8u162

**Comment by PrajaktaPurohit** [comment]
This will be done as a part of the next Chef-Server Release. Issue also mentioned in: https://github.com/chef/omnibus-software/issues/1082

**Comment by btm** [comment]
https://github.com/chef/chef-server/compare/praj/AdoptOpenJDK11

**Comment by tas50** [comment]
We moved away from Oracle JRE in the Chef Infra Server 14.0 release and have keep this up to date with each release of Infra Server since then. I'm going to close this issue out at this point as the Infra Server specific changes are complete.
